wurst/util/DialogBox.wurst

```diff
@@ -0,0 +1,84 @@
+package DialogBox
+import HashMap
+import Dialog
+import LinkedList
+
+public interface Callback
+    function run()
+
+/** DialogBox lets you to create dialogs and associate the buttons with code. */
+public class DialogBox
+
+    private static constant buttonClosures = new HashMap<button, Callback>
+    private static constant dialogBoxes = new HashMap<dialog, DialogBox>
+    private constant buttons = new LinkedList<button>
+
+    private constant dialogHandle = createDialog()
+    private constant clickDialog = CreateTrigger()
+
+    private string title    
+
+    construct(string title)
+        this.title = title
+        this.dialogHandle.setMessage(title)
+        dialogBoxes.put(this.dialogHandle, this)
+        this.clickDialog
+            ..registerDialogEvent(this.dialogHandle)
+            ..addAction(function removeClickedDialog)
+
+    private static function removeClickedDialog()
+        let btn = GetClickedButton()
+        let cb = buttonClosures.get(btn)
+        if(cb!=null)
+            cb.run()
+        destroy dialogBoxes.get(GetClickedDialog())
+
+    function addButton(string buttonText, Callback cb)
+        this.addButton(buttonText, 0 , cb)
+
+    /** Hotkey: use ASCII number of the capital letter. */
+    function addButton(string buttonText, integer hotkey, Callback cb)
+        let btn = this.dialogHandle.addButton(buttonText, hotkey)
+        this.buttons.add(btn)
+        buttonClosures.put(btn, cb)
+
+    /** Adds a quit button to this dialog. If it is clicked, it ends the game for that player. */
+    function addQuitButton(boolean doScoreScreen, string buttonText, Callback cb)
+        this.addQuitButton(doScoreScreen, buttonText, 0, cb)
+    
+    /** Adds a quit button to this dialog. If it is clicked, it ends the game for that player.
+    Hotkey: use ASCII numbers of the capital letter. */
+    function addQuitButton(boolean doScoreScreen, string buttonText, integer hotkey, Callback cb)
+        let btn = this.dialogHandle.addQuitButton(doScoreScreen, buttonText, hotkey)
+        this.buttons.add(btn)
+        buttonClosures.put(btn, cb)
+
+    /** Toggles visibility of the DialogBox for a player. Dialogs are invisible by default
+    Dialogs cannot be shown at map initialization */
```

Comment:
Add dot at the end of sentences, if that's how the whole package is documented. Sadly, we haven't yet made proper standards for how this is done (whether there's a dot or not), but having it consistent on a package level will make it easier to refactor later when standards are decided.